sdk/scheduler/src/test/java/com/mesosphere/sdk/offer/evaluate/NamedVIPEvaluationStageTest.java

```diff
@@ -45,4 +49,40 @@ public void testDiscoveryInfoPopulated() throws Exception {
         Assert.assertTrue(vipLabel.getKey().startsWith("VIP_"));
         Assert.assertEquals(vipLabel.getValue(), "test-vip:80");
     }
+
+    @Test
+    public void testVIPIsReused() throws InvalidRequirementException {
```

Comment:
Is a test needed which evaluates multiple offers?  I'm not grasping the bug exactly.